ansible_base/resource_registry/tasks/sync.py

```
@@ -73,6 +79,10 @@ class ManifestItem:
     resource_hash: str
     resource_data: dict | None = None
 
+    def __post_init__(self):
+          # Normalize to canonical UUID form; raises if malformed.
+          self.ansible_id = str(uuid.UUID(self.ansible_id))
```

Ruff (0.16.3)

[error] Undefined name `uuid`

(F821)